tools/dm/manage-task.md

```diff
@@ -0,0 +1,595 @@
+  update-task          update a task's config for routes, filters, column-mappings, black-white-list
+
+Flags:
+  -h, --help             help for dmctl
+  -w, --worker strings   DM-worker ID
+
+# 使用 `dmctl [command] --help` 来获取某个命令的更多信息
+```
+
+## 上游 MySQL 实例配置前置检查
+
+为提前检测出数据同步任务可能存在的一些配置错误，DM 提供了前置检查功能。`check-task` 命令可用于对上游 MySQL 实例配置是否满足 DM 要求进行前置检查。
+
+上下游数据库用户必须具备相应读写权限。当数据同步任务启动时，DM 会自动检查下列权限和配置：
+
++ 数据库版本
+
+    - 5.5 < MySQL 版本 < 5.8
+    - MariaDB 版本 >= 10.1.2
+
++ MySQL binlog 配置
+
+    - binlog 是否开启（DM 要求 binlog 必须开启）
+    - 是否有 `binlog_format=ROW`（DM 只支持 ROW 格式的 binlog 同步）
+    - 是否有 `binlog_row_image=FULL`（DM 只支持 `binlog_row_image=FULL`）
+
++ 上游 MySQL 实例用户的权限
+
+    DM 配置中的 MySQL 用户至少需要具备以下权限：
+
+    - REPLICATION SLAVE
+    - REPLICATION CLIENT
+    - RELOAD
+    - SELECT
+
++ 上游 MySQL 表结构的兼容性
+
+    TiDB 和 MySQL 的兼容性存在以下一些区别：
+
+    - TiDB 不支持外键
+    - 字符集的兼容性不同，详见 [TiDB 支持的字符集](/sql/character-set-support.md)
+
++ 上游 MySQL 多实例分库分表的一致性
+
+    + 所有分表的表结构是否一致，检查内容包括：
+
+        - Column 数量
+        - Column 名称
+        - Column 位置
+        - Column 类型
+        - 主键
+        - 唯一索引
+
+    + 分表中自增主键冲突检查
+
+        - 在两种情况下会造成检查失败：
+
+            - 分表存在自增主键，且自增主键 column 类型不为 bigint
+            - 分表存在自增主键，自增主键 column 类型为 bigint， 但没有为其配置列值转换
+
+        - 其他情况下检查将成功
+
+## 管理数据同步任务
+
+本部分描述了如何使用不同的任务管理命令来执行以下操作：
+
+- [创建数据同步任务](#创建数据同步任务)
+- [查询数据同步任务状态](#查询数据同步任务状态)
+- [暂停数据同步任务](#暂停数据同步任务)
+- [重启数据同步任务](#重启数据同步任务)
+- [停止数据同步任务](#停止数据同步任务)
+- [更新数据同步任务](#更新数据同步任务)
+
+### 创建数据同步任务
+
+`start-task` 命令用于创建数据同步任务。 当数据同步任务启动时，DM 将[自动对相应权限和配置进行前置检查](#上游-MySQL-实例配置前置检查)。
+
+```bash
+» help start-task
+start a task with config file
+
+Usage:
+ dmctl start-task [-w worker ...] <config_file> [flags]
+
+Flags:
+ -h, --help   help for start-task
+
+Global Flags:
+ -w, --worker strings   dm-worker ID
+```
+
+#### 命令用法示例
+
+```bash
+start-task [ -w "172.16.30.15:10081"] ./task.yaml
+```
+
+#### 参数解释
+
++ `-w`：
+    - 可选
+    - 指定在特定的一组 DM-workers 上执行 `task.yaml`
+    - 如果设置，则只启动指定任务在该组 DM-workers 上的子任务
++ `config_file`：
+    - 必选
+    - 指定 `task.yaml` 的文件路径
+
+#### 返回结果示例
+
+```bash
+{
+​    "result": true,
+​    "msg": "",
+​    "workers": [
+​        {
+​            "result": true,
+​            "worker": "172.16.30.15:10081",
+​            "msg": ""
+​        },
+​        {
+​            "result": true,
+​            "worker": "172.16.30.16:10081",
+​            "msg": ""
+​        }
+​    ]
+}
+```
+
+### 查询数据同步任务状态
+
+`query-status` 命令用于查询数据同步任务状态。有关查询结果及子任务状态，详见[查询状态](/tools/dm/query-status.md)。
+
+```bash
+» help query-status
+query task's status
+
+Usage:
+ dmctl query-status [-w worker ...] [task_name] [flags]
+
+Flags:
+ -h, --help   help for query-status
+
+Global Flags:
+ -w, --worker strings   dm-worker ID
+```
+
+#### 命令用法示例
+
+```bash
+query-status
+```
+
+#### 参数解释
+
+- `-w`：
+    - 可选
+    - 查询在指定的一组 DM-workers 上运行的数据同步任务的子任务
+- `task_name`：
+    - 可选 
+    - 指定任务名称 
+    - 如果未设置，则返回全部数据同步任务的查询结果
+
+#### 返回结果示例
```

Comment:
for `query-status`, we have a doc with comments for response fields, should we link to it?